src/main/java/g2001_2100/s2079_watering_plants/readme.md

```diff
@@ -62,13 +62,13 @@ Steps needed = 3 + 3 + 4 + 4 + 5 + 5 + 6 = 30.
 
 **Explanation:**
 
-You have to refill before watering each plant. 
+You have to refill before watering each plant.
 
 Steps needed = 1 + 1 + 2 + 2 + 3 + 3 + 4 + 4 + 5 + 5 + 6 + 6 + 7 = 49.
 
 **Constraints:**
 
 * `n == plants.length`
 * `1 <= n <= 1000`
-* `1 <= plants[i] <= 10<sup>6</sup>`
-* `max(plants[i]) <= capacity <= 10<sup>9</sup>`
+* `1 <= plants[i] <= 10⁶`
```

Comment:
It should be code tag.

Generatemd tool https://github.com/javadev/LeetCode-in-Java/releases/download/generatemd/generatemd-1.0.jar